charts/pulsar/templates/function-worker/function-worker-role-binding.yaml

```diff
@@ -0,0 +1,68 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+{{- if not .Values.functions.serviceAccount.clusterRole }}
+{{- if and (and .Values.components.functions .Values.functions.useDedicateRunner) .Values.functions.serviceAccount.create }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.functions.component }}-rolebinding"
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.functions.component }}-role"
+subjects:
+- kind: ServiceAccount
+  name: {{ template "pulsar.function.serviceAccount" . }}
+  namespace: {{ template "pulsar.namespace" . }}
+---
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: "{{ template "pulsar.fullname" . }}-{{ .Values.functions.component }}-role"
+  labels:
+    {{- include "pulsar.standardLabels" . | nindent 4 }}
+rules:
+- apiGroups: [""]
+  resources:
+  - configmap
+  - configmaps
+  verbs: ["get", "list", "watch"]
+- apiGroups: ["", "extensions", "apps"]
+  resources:
+    - pods
```

Comment:
does it need access to `configmap?`

Reply:
seems we don't need configmap permission, removed.

Reply:
somehow seems the document said configmap permission is needed, I think better just keep it: https://pulsar.apache.org/docs/en/functions-runtime/

Reply:
I believe we just need `get`, `list`, and `watch` permission on configmap. But that should have been covered by https://github.com/streamnative/charts/pull/426/files#diff-437f94e036fc125552f2cb0f10309621cbd593de448938ff6bfbb332964cc6a8R45